cli.py

```diff
@@ -76,6 +80,14 @@ def setup_pipenv(component_project_dir: Path):
     else:
         raise FileNotFoundError(f"{pipfile} is not a file")
 
+def create_env_file(component_project_dir: Path):
+    envfile = component_project_dir / ".env"
+    if not envfile.exists():
+        print(f"creating {envfile}")
+        with open(envfile, 'w') as envfile_content:
+            envfile_content.write(ENVFILE_TEMPLATE)
+    else:
+        raise FileNotFoundError(f"{envfile} is not a file")
```

Comment:
This error is raised, if there is already the .env file. The message is misleading.
How about f"Cannot create config file {envfile} . Name already in use."